src/main/java/com/hobbycollection/HobbyCollectionRESTController.java

```diff
@@ -8,6 +8,8 @@
 @RestController
 public class HobbyCollectionRESTController {
 
+    //comment about what the method does
+
```

Comment:
Putting conversation/discussion  in code comments is bad practice, as that clutters up the source code and does not belong in source code files.  I mentioned that in the code review instructions.    Discussion and comments are very good, but have that in the appropriate place, which is in the "Files Changed" tab in the pull request in GitHub (which is where I am writing this and you are reading this).  That way, you can have a conversation with replies, right where the code is happening, and without cluttering up the source code itself.